core/src/main/scala/org/apache/spark/scheduler/WorkerOffer.scala

```diff
@@ -21,4 +21,8 @@ package org.apache.spark.scheduler
  * Represents free resources available on an executor.
  */
 private[spark]
-case class WorkerOffer(executorId: String, host: String, cores: Int)
+case class WorkerOffer(
+    executorId: String,
+    host: String,
+    cores: Int,
+    address: Option[String] = None)
```

Comment:
What is the difference between host and address? It is useful to mention it in the doc.